openssl-dynamic/src/main/c/ssl.c

```diff
@@ -803,14 +803,7 @@ TCN_IMPLEMENT_CALL(jlong /* SSL * */, SSL, newSSL)(TCN_STDARGS,
     int *handshakeCount = NULL;
     tcn_ssl_ctxt_t *c = J2P(ctx, tcn_ssl_ctxt_t *);
 
-    if (c == NULL) {
-        tcn_ThrowException(e, "ssl ctx is null");
-        return 0;
-    }
-    if (c->ctx == NULL) {
-        tcn_ThrowException(e, "ctx is null");
-        return 0;
-    }
+    TCN_CHECK_NULL(c, "ctx is null", 0);
```

Comment:
This does not work an will not compile. If the method signature returns a `jint` or `jlong` I can not return NULL